### ISSUE TITLE
signblock fails with dynamic blocks missing witnessScript error

### ISSUE THREAD

**Issue by wintercooled** [ElementsProject/elementsproject.github.io#105]
The line...

```
SIGN1=$(e1-cli signblock $HEX)
```

fails with error 8 - "Signing dynamic blocks requires the witnessScript argument"

This is a change in 0.18 release relating to dynamic blocks.

Looking at fixing this now...

**Comment by wintercooled** [comment]
Can be fixed with:

```
@phil, replacing the setting of the SIGNBLOCKARG variable to the following line will get you past the error you had:

```
SIGNBLOCKARG="-signblockscript=$(echo $REDEEMSCRIPT) -con_max_block_sig_size=150 -con_dyna_deploy_start=0"
```
```

**Comment by wintercooled** [comment]
Fixed with #106
